content/en/docs/contributing/localization.md

```diff
@@ -18,22 +18,44 @@ guidance offered in this section.
 
 ### Summary / TL;DR {#summary}
 
-Do translate:
+#### ✅ Do {#do}
 
-- [Front matter][] fields `title`, `linkTitle`, and `description`
-- Page content, including the text fields in Mermaid [diagrams](#images)
+<div class="border-start border-success bg-success-subtle">
 
-Do **NOT**:
+- **Translate**:
+  - Page content, including:
+    - Mermaid [diagram](#images) text fields
+  - [Front matter][] fields `title`, `linkTitle`, and `description`
+  - **All** page content and front matter unless indicated otherwise
```

Comment:
Isn't this redundant?

Reply:
I call out explicitly that _all_ content should be translated.
And brings to light that sometimes the English pages have notes stating that some content (mainly front-matter fields atm), _should **not**_ be translated or included.